tox.ini

```diff
@@ -68,21 +68,20 @@ deps =
     online: astroquery
     hypothesis: astroquery
     # Figure tests need a tightly controlled environment
-    figure-!devdeps: astropy==5.1.0
-    figure-!devdeps: matplotlib==3.5.2
+    figure-!devdeps: astropy==5.2.0
+    figure-!devdeps: matplotlib==3.6.2
     figure-!devdeps: mpl-animators==1.0.0
-    figure-!devdeps: numpy<1.24
```

Comment:
Again, I think it's worth keeping this as the minimum version we support.

Reply:
The base figure tests do not test for min versions, this is not the earliest version we use. This was pinned to avoid the issue with the deprecation warning.